portal-impl/src/portal.properties

```diff
@@ -6919,6 +6924,11 @@
     #
     feature.flag.LPD-6378.system=true
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASED__MINUS__NUMBER1__NUMBER0__NUMBER7__NUMBER0__NUMBER3__PERIOD_TYPE
+    #
+    feature.flag.LPD-10703.type=release
```

Comment:
Don't set the type yet since this feature is still WIP. Dev FF are hidden from the UI by default, so we usually start with dev FF and update it to the desired type after the epic is ready for release.